app/views/checkout/thankyou.scala.html

```diff
@@ -13,11 +13,18 @@
 
 @main("Confirmation | Digital | Subscriptions | The Guardian", bodyClasses=List("is-wide"), touchpointBackendResolutionOpt = Some(touchpointBackendResolution)) {
 
-<main class="page-container gs-container gs-container--slim"
-data-tracking-prop17="GuardianDigiPack:Order Complete"
-data-tracking-products="Subscriptions and Membership;GUARDIAN_DIGIPACK;@{subscriptionProduct.frequency.numberOfMonths};@{subscriptionProduct.price};purchase">
->
-
+<script type="text/javascript">
+    guardian.slug="GuardianDigiPack:Order Complete";
```

Comment:
Im not sure I agree, I think this is view specific stuff and would rather not have it in the controller but I see there is similar with the JSVars so ill stick with it for consistency.